sway-lib-std/src/u256.sw

```diff
@@ -23,16 +23,24 @@ fn rsh_with_carry(word: u64, shift_amount: u64) -> (u64, u64) {
 }
 
 /// The 256-bit unsigned integer type.
+///
+/// # Additional Information
+///
 /// Represented as four 64-bit components: `(a, b, c, d)`, where `value = (a << 192) + (b << 128) + (c << 64) + d`.
 pub struct U256 {
+    /// The first 64-bit component of the `U256`.
```

Comment:
Maybe something like "The first 64-bit component of the `U256`, holding the most significant bits"?